content/projects/gsoc/2020/project-ideas/git-plugin-caching.adoc

```diff
@@ -31,4 +31,28 @@ Stephen Connolly recommends that the capability be created as a separate plugin
 That would avoid the risk of users seeing this new feature without intentionally installing a new plugin specifically for the feature.
 If a separate plugin is the path that is chosen, then Mark Waite will need additional help from other mentors that are more familiar with the Jenkins extension mechanisms.
 
+=== Quickstart
+
+The project idea is expected to require changes in both the git client plugin and the git plugin.
+
+Install a git client plugin development environment by following the link:https://github.com/jenkinsci/git-client-plugin/blob/master/CONTRIBUTING.adoc[contributing instructions].
+Compile the plugin, run its automated tests, and confirm that the automated tests are passing.
+Enable coverage reporting and review the coverage report.
+
+Install a git plugin development environment by following the link:https://github.com/jenkinsci/git-plugin/blob/master/CONTRIBUTING.adoc[contributing instructions].
+Compile the plugin, run its automated tests, and confirm that the automated tests are passing.
+Enable coverage reporting and review the coverage report.
+
+Install an integrated development environment (Netbeans, Eclipse, IntelliJ, ...).
+Run the git plugin in the development environment.
+Set a breakpoint and confirm that the breakpoint is reached in the development environment.
+
+Consider implementing a fix for one of the link:https://issues.jenkins-ci.org/issues/?jql=(component%3Dgit-plugin%20OR%20component%20%3D%20git-client-plugin)%20and%20labels%20%3D%20newbie-friendly%20and%20status%20in%20(Open%2CReopened)[newbie friendly issues].
+
+=== Links
+
+* link:https://groups.google.com/d/msg/jenkinsci-gsoc-all-public/n3djkbJsN_8/hifQN2RvCQAJ[Jenkins GSoC mailing list discussion of repository caching idea]
+
+=== Google Doc Draft Idea
```

Comment:
I suggest removing it, the link is generated automatically by the layout page

Reply:
Thanks.  Done in 172a2ffa1d596af4c1d62ed4778534b8baac27ca